### PR TITLE
Validation: Added Unique Descriptions

### PR DESCRIPTION
**Pull request by JustPies**
Fixed:

- description-missing errors

This PR fixes 30 issues across 30 files and is part of ongoing Validation cleanup efforts. No other content has been edited or added.

### CODE REVIEW

File: dotnet-desktop-guide/framework/winforms/controls/walkthrough-authoring-a-composite-control-with-visual-csharp.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ helpviewer_keywords:
   - "user controls [C#]"
   - "custom controls [Windows Forms], creating"
 ms.assetid: f88481a8-c746-4a36-9479-374ce5f2e91f
+description: Learn how to author a composite control with c# in order to provide a means by which custom graphical interfaces can be created and reused.
```
Comment (mSarts) [inline dotnet-desktop-guide/framework/winforms/controls/walkthrough-authoring-a-composite-control-with-visual-csharp.md:13]: C# should be capitalized.

**Comment by JustPies** [comment]
Global effort to fix validation errors

@adegeo

The Content & Learning team is fixing build validation errors on docs.microsoft.com. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. I’d very much appreciate your timely review and feedback on these changes I’ve suggested. Thanks!

File: dotnet-desktop-guide/framework/winforms/controls/user-drawn-controls.md [hunk 2]
```diff
@@ -9,6 +9,7 @@ helpviewer_keywords:
   - "OnPaint method [Windows Forms]"
   - "user-drawn controls [Windows Forms]"
 ms.assetid: 034af4b5-457f-4160-a937-22891817faa8
+description: Learn how the .NET Framework provides the functionality to create a user control or one that inherits from an existing control.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/winforms/controls/user-drawn-controls.md:12]:
```suggestion
description: Learn how Windows Forms provides the functionality to create a user control or one that inherits from an existing control.
```

File: dotnet-desktop-guide/framework/winforms/controls/using-the-managed-html-document-object-model.md [hunk 3]
```diff
@@ -4,6 +4,7 @@ ms.date: "03/30/2017"
 helpviewer_keywords: 
   - "managed HTML DOM"
 ms.assetid: a017dd5c-cd7b-47e4-952c-f4f54cb48409
+description: Learn how the managed HTLM document object model provides a wrapper based on the .NET Framework for the HTML classes exposed by Internet Explorer.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/winforms/controls/using-the-managed-html-document-object-model.md:7]:
```suggestion
description: Learn how the managed HTML document object model provides a wrapper based on the .NET Framework for the HTML classes exposed by Internet Explorer.
```

**Review by adegeo — APPROVED** [review]

**Pull request merged**